### DISCUSSION TITLE
The flow calibration with a nozzle of 0.8 does not work correctly

### DISCUSSION THREAD

**Discussion by DrawdeRus** [OrcaSlicer/OrcaSlicer#5665] [General]
https://github.com/SoftFever/OrcaSlicer/assets/127427251/973cbd34-508d-49e2-93af-ab192dabc75b

**Comment by otede** [comment]
I think I saw that bug fixed somewhere. You might want to post this as an issue. Maybe there's some other variable in play than in case of the fixed case.
